### ISSUE TITLE
Build failure: python3Packages.hvplot

### ISSUE THREAD

**Issue by n8henrie** [NixOS/nixpkgs#360786]
## Steps To Reproduce

aarch64-darwin

```
nix build github:nixos/nixpkgs/3b5e1da6b9652d80c9b487bb715484c88eacfb7c#python3Packages.hvplot
```

## Build log

https://gist.github.com/n8henrie/8a5d476e38ac77b194609b8623314f5d

## Metadata

 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.1.0, macOS 15.1.1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - channels(n8henrie): `""`
 - channels(root): `""`
 - nixpkgs: `/nix/store/3ljzzvxvvc4galpxpxrpz9xv916j9639-source`

## Notify maintainers

None listed -- looks like @costrouc created the package

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by FliegendeWurst** [comment]
I think the log was truncated, or maybe I'm just unable to spot the error.

```
@nix { "action": "setPhase", "phase": "fixupPhase" }
checking for references to /private/tmp/nix-build-python3.12-hvplot-0.11.1.drv-0/ in /nix/store/qxzf79w5jrds6w2azd6xfqk6q8r3nv9y-python3.12-hvplot-0.11.1...
patching script interpreter paths in /nix/store/qxzf79w5jrds6w2azd6xfqk6q8r3nv9y-python3.12-hvplot-0.11.1
stripping (with command strip and flags -S) in  /nix/store/qxzf79w5jrds6w2azd6xfqk6q8r3nv9y-python3.12-hvplot-0.11.1/lib
checking for references to /private/tmp/nix-build-python3.12-hvplot-0.11.1.drv-0/ in /nix/store/a496vm8xfl9s6ixsa0y4z9gdrpwndj6a-python3.12-hvplot-0.11.1-dist...
patching script interpreter paths in /nix/store/a496vm8xfl9s6ixsa0y4z9gdrpwndj6a-python3.12-hvplot-0.11.1-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
```

**Comment by paparodeo** [comment]
looks like it is a headless issue. needs same fix as https://github.com/NixOS/nixpkgs/pull/350784/commits/9f3e8a9f0b97939c5cc79f98842db3b7940954ed

**Comment by n8henrie** [comment]
@FliegendeWurst -- sorry, not entirely sure why that happened -- there was a segfault in the logs locally. GitHub complained that it was too long to embed into the issue, so I piped directly to `gh gist create` but didn't look through the final output carefully. I find that builds sometimes (inappropriately) succeed the second time if the failure is only in `checkPhase` (and if I don't specify `--rebuild`) -- perhaps that's the issue?. I also ran it through a "decolorizer" (https://github.com/NixOS/nix/pull/6911#issuecomment-2137707474), but I think that's unlikely to be the culprit. Seems to be fixed in the PR regardless.

Thanks @paparodeo -- glad you recognized that.
